java-lotto/src/main/java/domain/Lotto/LottoService.java

```diff
@@ -17,11 +17,14 @@ public List<LottoTicket> generateLottoTickets(int count) {
         return tickets;
     }
 
-    public LottoResult calculateResults(List<LottoTicket> tickets, LottoTicket winningTicket) {
-        Map<Integer, Integer> matchCounts = new HashMap<>();
+    public LottoResult calculateResults(List<LottoTicket> tickets, LottoTicket winningTicket, int bonusNumber) {
```

Comment:
```suggestion
    public LottoResult calculateResults(List<LottoTicket> tickets, WinningTicket) {
```

로또 번호와 보너스 번호를 조합으로 묶은 새로운 모델을 만들어봐도 좋을 것 같아요
`WinningTicket`만의 책임들이 무엇인지도 고민해 보면 더 좋을 것 같구요! 👍